app/addons/activetasks/actions.js

```diff
@@ -14,13 +14,14 @@ define([
   'addons/activetasks/actiontypes',
   'addons/activetasks/resources'
 ],
-function (FauxtonAPI, ActionTypes, Resources) {
+function (FauxtonAPI, ActionTypes, Resources, Actions) {
```

Comment:
What do we need the Actions param for? (at least it does not match with above, and we _are_ w/in actions...